tests/page/page-network-response.spec.ts

```diff
@@ -18,6 +18,7 @@
 import url from 'url';
 import { test as it, expect } from './pageTest';
 import fs from 'fs';
+import exp from 'constants';
```

Comment:
What is this? :)

Reply:
I wish I knew, vs code adding something behind my back:) Removed.